### PR TITLE
Translation update

### PR DESCRIPTION
**Pull request by harambasicluka**
All translation changes are collected in this PR.

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,1028 @@
...
+    <!-- XHED: settings - settings overview page title -->
+    <string name="settings_title">"Настройки"</string>
+    <!-- XTXT: settings - on, like a label next to a setting -->
+    <string name="settings_on">"Вкл."</string>
+    <!-- XTXT: settings - off, like a label next to a setting -->
+    <string name="settings_off">"Выкл."</string>
+    <!-- XHED: settings(tracing) - page title -->
+    <string name="settings_tracing_title">"Определение контакта"</string>
+    <!-- XHED: settings(tracing) - headline bellow illustration -->
+    <string name="settings_tracing_headline">"Вот как работает определение контакта "</string>
```
Comment (MichaelPFrey) [inline Corona-Warn-App/src/main/res/values-ru/strings.xml:414]:
```suggestion
    <string name="settings_tracing_headline">"Вот как работает определение контакта"</string>
```
?

as neither `de` nor `en` end on a white space:
https://github.com/corona-warn-app/cwa-app-android/blob/fccc32a84f2349631a08d09da5f751bce8f94ee3/Corona-Warn-App/src/main/res/values-de/strings.xml#L415
https://github.com/corona-warn-app/cwa-app-android/blob/fccc32a84f2349631a08d09da5f751bce8f94ee3/Corona-Warn-App/src/main/res/values-en/strings.xml#L415

File: Corona-Warn-App/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -0,0 +1,1028 @@
...
+    <!-- XTXT: Path to the full blown privacy html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_privacy_html_path">"privacy_ru.html"</string>
+    <!-- XHED: Page title for terms of use information page, also menu item / button text -->
+    <string name="information_terms_title">"Условия использования"</string>
+    <!-- XHED: Page headline for terms of use information page -->
+    <string name="information_terms_headline">"Содержимое"</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_terms_illustration_description">"Один человек держит устройство с длинным текстом на экране. Рядом с текстом отображается галочка, которую нужно установить для принятия условий использования."</string>
+    <!-- XTXT: Path to the full blown terms html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_terms_html_path">"terms_ru.html"</string>
```
Comment (MichaelPFrey) [inline Corona-Warn-App/src/main/res/values-ru/strings.xml:562]: shouldn't that either
   * link to an existing file (https://github.com/corona-warn-app/cwa-app-android/tree/dev/Corona-Warn-App/src/main/assets)

or

   * terms_ru.html

be part of this pull request?

File: Corona-Warn-App/src/main/res/values-ru/strings.xml [hunk 3]
```diff
@@ -0,0 +1,1028 @@
...
+    <string name="information_about_body">"Каждый пользователь приложения помогает отслеживать и останавливать передачу инфекции. Приложение регистрирует контакты с другими людьми локально на вашем устройстве. Вы получаете уведомления о контактах с людьми, которые впоследствии получили положительный результат теста на COVID-19. Ваши персональные данные и конфиденциальность надежно защищены."</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_about_illustration_description">"Группа людей использует свои устройства в городе."</string>
+    <!-- XHED: Page title for privacy information page, also menu item / button text -->
+    <string name="information_privacy_title">"Конфиденциальность данных"</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_privacy_illustration_description">"На устройстве одной женщины установлено приложение Corona-Warn-App. Значок замка на щите указывает на шифрование данных."</string>
+    <!-- XTXT: Path to the full blown privacy html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_privacy_html_path">"privacy_ru.html"</string>
```
Comment (MichaelPFrey) [inline Corona-Warn-App/src/main/res/values-ru/strings.xml:554]: shouldn't that either
   * link to an existing file (https://github.com/corona-warn-app/cwa-app-android/tree/dev/Corona-Warn-App/src/main/assets)

or

   * privacy_ru.html

be part of this pull request?

File: Corona-Warn-App/src/main/res/values-ar/strings.xml [hunk 4]
```diff
@@ -0,0 +1,1024 @@
...
+    <!-- ####################################
+                    Main
+    ###################################### -->
+
+    <!-- XHED: main, FAQ -->
+    <string name="main_about_headline">"الأسئلة المتكررة"</string>
+    <!-- XTXT: main, explains faq on card -->
+    <string name="main_about_body">"يمكنك أن تجد هنا إجابات الأسئلة المتكررة حول تطبيق Corona-Warn-App. سيتم توجيهك إلى موقع إلكتروني خارجي."</string>
+    <!-- XTXT: FAQ link, should be translated  -->
+    <string name="main_about_link">"https://www.bundesregierung.de/corona-warn-app-faq"</string>
```
Comment (MichaelPFrey) [inline Corona-Warn-App/src/main/res/values-ar/strings.xml:188]: there is an Arabic FAQ: https://www.bundesregierung.de/breg-de/themen/corona-warn-app/corona-warn-app-arabisch

so maybe
```suggestion
    <string name="main_about_link">"https://www.bundesregierung.de/breg-de/themen/corona-warn-app/corona-warn-app-arabisch"</string>
```

File: Corona-Warn-App/src/main/res/values-ar/strings.xml [hunk 5]
```diff
@@ -0,0 +1,1024 @@
...
+    <!-- YTXT: Body text for about information page -->
+    <string name="information_about_body">"يمكن لأي شخص يستخدم التطبيق تتبع سلسلة العدوى وكسرها. يحفظ التطبيق التجمُّعات مع الأشخاص الآخرين محليًا على جهازك. ويتم إشعارك عند تجمُّعك مع أشخاص تم تشخيصهم مؤخرًا على أنهم مصابون بمرض كوفيد-19، علمًا أن هويتك وخصوصيتك محميتان دائمًا."</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_about_illustration_description">"مجموعة أشخاص يستخدمون أجهزتهم في المدينة جميعًا."</string>
+    <!-- XHED: Page title for privacy information page, also menu item / button text -->
+    <string name="information_privacy_title">"خصوصية البيانات"</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_privacy_illustration_description">"إحدى السيدات تستخدم تطبيق Corona-Warn-App على جهازها. القفل الموجود على الشاشة هو رمز للبيانات المشفرة."</string>
+    <!-- XTXT: Path to the full blown privacy html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_privacy_html_path">"privacy_en.html"</string>
```
Comment (MichaelPFrey) [inline Corona-Warn-App/src/main/res/values-ar/strings.xml:550]: Just a friendly reminder to point out a potential inconsistency:
Linking to `privacy_en.html` makes sense, when their is no full translation (yet),
but note that in the `ru` translation in this PR, this links is updated to `privacy_ru.html`.

File: Corona-Warn-App/src/main/res/values-ar/strings.xml [hunk 6]
```diff
@@ -0,0 +1,1024 @@
...
+    <!-- XHED: Page title for terms of use information page, also menu item / button text -->
+    <string name="information_terms_title">"شروط الاستخدام"</string>
+    <!-- XHED: Page headline for terms of use information page -->
+    <string name="information_terms_headline">"المحتوى"</string>
+    <!-- XACT: describes illustration -->
+    <string name="information_terms_illustration_description">"يحمل شخص ما جهازًا به الكثير من النصوص على الشاشة. بجوار النص توجد علامة صح تمثل رمزًا لقبول شروط الاستخدام."</string>
+    <!-- XTXT: Path to the full blown terms html, to translate it exchange "_de" to "_en" and provide the corresponding html file -->
+    <string name="information_terms_html_path">"terms_en.html"</string>
```
Comment (MichaelPFrey) [inline Corona-Warn-App/src/main/res/values-ar/strings.xml:558]: Just a friendly reminder to point out a potential inconsistency:
Linking to `terms_en.html` makes sense, when their is no full translation (yet),
but note that in the `ru` translation in this PR, this links is updated to `terms_ru.html`.

File: Corona-Warn-App/src/main/res/values-ru/strings.xml [hunk 7]
```diff
@@ -0,0 +1,1028 @@
...
+    <!-- XBUT: settings(reset) - dialog confirm -->
+    <string name="settings_reset_dialog_button_confirm">"Сброс"</string>
+    <!-- XHED: settings(background priority) - notification page title -->
+    <string name="settings_background_priority_title">"Priorisierte Hintergrundaktivität"</string>
+    <!-- XTXT: settings(background priority) - text in row on settings overview -->
+    <string name="settings_background_priority_body_description">"Erlauben Sie die automatische Aktualisierung Ihres Risikostatus."</string>
+    <!-- XHED: settings(background priority) - multiline headline below illustration -->
+    <string name="settings_background_priority_headline">"Corona-Warn-App im Hintergrund ausführen"</string>
+    <!-- YTXT: settings(background priority) - description text -->
+    <string name="settings_background_priority_body">"Wenn Sie die Priorisierte Hintergrundaktivität anschalten, wird die Corona-Warn-App dauerhaft im Hintergrund ausgeführt. So kann die App jederzeit Ihren Risikostatus ermitteln."</string>
```
Comment (MichaelPFrey) [inline Corona-Warn-App/src/main/res/values-ru/strings.xml:521]: I assume the strings above will be translated later?

**Review by pwoessner — DISMISSED** [review]

**Review by pwoessner — APPROVED** [review]

**Pull request merged**